test/acceptance/features/bindAppToService.feature

```diff
@@ -753,3 +830,49 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-cross-ns-service" is ready
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    Scenario: Binding Secret is updated externally
+        Given OLM Operator "backend-new-spec" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+            spec:
+                host: example.common
+            """
+        * Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-secret-updated-externally
+            spec:
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+            """
+        Then Secret name should be updated in Service Binding status
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "False"
+        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "True"
+        And Secret contains "BACKEND_HOST" key with value "example.common"
+
+        * Binding secret is updated
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: binding-request-secret-updated-externally
+            type: Opaque
+            stringData:
+                host2: example2.common
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "False"
+        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-secret-updated-externally" should be changed to "True"
+        And Secret contains "BACKEND_HOST" key with value "example.common"
+        And Secret does not contain "BACKEND_HOST2"
```

Comment:
and on top of these checks, we should assert that the secret content got reverted by the operator.

Reply:
Added check `And Secret does not contain "host2"` would mean that the old content got reverted